pulp_rpm/app/depsolving.py

```diff
@@ -669,18 +669,11 @@ def _load_from_version(self, repo_version, as_target=False):
             "pk"
         )
 
-        nonmodular_rpms = models.Package.objects.filter(
+        packages = models.Package.objects.filter(
             pk__in=package_ids, is_modular=False
```

Comment:
in that case we should load all rpms regardless of the modular flag, don't we?

Reply:
whoops, missed that.  yes.